app/controllers/api/host_aggregates_controller.rb

```diff
@@ -25,6 +25,34 @@ def delete_resource_main_action(_type, host_aggregate, _data = {})
       {:task_id => host_aggregate.delete_aggregate_queue(current_user.userid)}
     end
 
+    def add_host_resource(type, id, data = {})
```

Comment:
I didn't want to muddy up this PR before it was merged, but @kbrock is this a candidate for enqueue_ems_action?